src/js/containers/generateEF/GenerateEFContainer.jsx

```diff
@@ -33,14 +35,28 @@ class GenerateEFContainer extends React.Component {
 			hasErrors: false,
 			e: {},
 			f: {},
-			generated: false
+			generated: false,
+			gtas: null
 		};
 
 	}
 
 	componentDidMount() {
 		this.isUnmounted = false;
 		this.checkFileStatus();
+		this.isGtas();
+	}
+
+	isGtas() {
+		ReviewHelper.isGtasWindow()
+			.then((res) => {
+				if(res.open != this.state.gtas) {
+					this.setState({gtas: res})
```

Comment:
Since all we're passing back now is the "data" element in the JSON (which is either `null` or an object) it might make sense to set gtas to `res.data`)